Improvement/MC-11609 Add fields missing and routes doc (#182)
cloudops/cloudmc-api-docs: source/includes/gcp/_routes.md

### Routes

> A route is a paths that network traffic takes from an [instances](#gcp-instances) to other destinations. The destination can be inside a GCP network or outside.

Reviewer: ```suggestion
A route is a path that network traffic takes from an [instance](#gcp-instances) to other destinations. The destination can be inside a GCP network or outside.
```

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

> A route is a paths that network traffic takes from an [instances](#gcp-instances) to other destinations. The destination can be inside a GCP network or outside.

Reviewer 2: Should this be "a route is a path"?

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

      "description": "Default local route to the subnetwork 10.172.0.0/20.",
      "network": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
      "destRange": "10.172.0.0/20",
      "priority": 0,
      "nextHopNetwork": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/routes/default-route-0f2f9d9f9910829a",
      "kind": "compute#route",
      "networkName": "default"
    }],
  "metadata": {
    "recordCount": 1
  }
}
```

<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/routes</code>

Retrieve a list of all routes in an [environment](#administration-environments).

Attributes | &nbsp;
---------- | -----
`id`<br/>*string* | The route's id.
`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
`name`<br/>*string* | The route's name.
`description`<br/>*string* | An optional description of this resource.
`network`<br/>*string* | URL of the network to which this route is attached.

> `destRange`<br/>*string* | The route's address destinatiation.

Reviewer 2: Typo in destination

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

`priority`<br/>*integer* | Priority for this route. Number should be a positive integer. Priority is used to break ties when there is more than one matching route of maximum length.

> `nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway

Reviewer: ```suggestion
`nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway.
```

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

> `nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway

Reviewer 2: Missing a period here

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

    "destRange": "10.172.0.0/20",
    "priority": 0,
    "nextHopNetwork": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
    "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/routes/default-route-0f2f9d9f9910829a",
    "kind": "compute#route",
    "networkName": "default"
  }
}
```

<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/routes/:id</code>

Retrieve a route in a given [environment](#administration-environments).

Attributes | &nbsp;
---------- | -----
`id`<br/>*string* | The route's id.
`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
`name`<br/>*string* | The route's name.
`description`<br/>*string* | An optional description of this resource.
`network`<br/>*string* | URL of the network to which this route is attached.
`destRange`<br/>*string* | The route's address destinatiation.
`priority`<br/>*integer* | Priority for this route. Number should be a positive integer. Priority is used to break ties when there is more than one matching route of maximum length.

> `nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway

Reviewer: ```suggestion
`nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway.
```

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7

> `nextHopNetwork`<br/>*string* | The next hop handles the matching packets for this route. It can be an instance, an IP address or the default internet gateway

Reviewer 2: Same here, missing period

Author: 1c902795ba00f6ba1f4af8f97b9112267ef3e8a7
